server/models/builtin-tasks.js

```diff
@@ -389,6 +389,7 @@ const anomalyDetectionTask = {
     name: 'Anomaly detection',
     description: 'Task will go through the signal set finding anomalies based on the specified parameters.',
     type: TaskType.PYTHON,
+    subType: PythonSubtypes.ANOMALY,
```

Comment:
I have to make sure this works and correctly has the packages that anomaly depends on installed.